lib/main.js

```diff
@@ -84,20 +84,19 @@ function get(uri) {
 module.exports = class AssetPipeClient {
     constructor({
         serverId,
+        server,
         buildServerUri,
```

Comment:
Is `buildServerUri` replaced by `server`?

Reply:
Yup. Probably should have mentioned that.